src/components/molecules/delimited-list/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import Tooltip from "../../atoms/tooltip"
+
+type DelimitedListProps = {
+  list: Record<string, any>[]
+  delimit?: number
+  property?: string
+}
+
+const DelimitedList: React.FC<DelimitedListProps> = ({
+  list,
+  delimit = 1,
+  property = "name",
+}) => {
+  if (!list.length) {
+    return <></>
+  }
+
+  const ToolTipContent = () => {
+    return (
+      <div className="flex flex-col">
+        {list.slice(delimit).map((listItem) => (
+          <span key={listItem.id}>{listItem[property]}</span>
+        ))}
+      </div>
+    )
+  }
+
+  return (
+    <span className="inter-small-regular">
+      {list
```

Comment:
suggetion: assign the sliced content to a const and create a boolean const such as `hasMore` with the constraint `list.length > delimit`